### PR TITLE
Update Flutter Mobile Vitals metrics links

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Fixes broken links due to a previous update where I had copied and pasted the Android metrics to the Flutter metrics table.

### Motivation
I was in the middle of consolidating the mobile vitals pages when I noticed the links were broken.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by xgouchet — APPROVED** [review]

File: content/en/real_user_monitoring/flutter/mobile_vitals.md [hunk 1]
```diff
@@ -39,18 +39,25 @@ The following metrics provide insight into your mobile application's performance
...
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /real_user_monitoring/ios/mobile_vitals/
 [2]: /real_user_monitoring/android/mobile_vitals/
-[3]: https://docs.flutter.dev/perf/best-practices#control-build-cost
-[4]: https://docs.flutter.dev/perf/ui-performance#identifying-problems-in-the-gpu-graph
+[3]: https://docs.flutter.dev/perf/ui-performance
+[4]: https://docs.flutter.dev/tools/devtools/performance
+[5]: https://developer.android.com/topic/performance/vitals/frozen [ NEED TO REPLACE WITH FLUTTER DOC LINK IF STILL RELEVANT ]
```
Comment (fuzzybinary) [inline content/en/real_user_monitoring/flutter/mobile_vitals.md:56]: I think it's fine to use the Android link here. It's the only documentation that I know of that explains this (iOS uses the Android documentation for this as well).

File: content/en/real_user_monitoring/flutter/mobile_vitals.md [hunk 2]
```diff
@@ -39,18 +39,25 @@ The following metrics provide insight into your mobile application's performance
...
+[6]: https://developer.android.com/topic/performance/vitals/anr [ NEED TO REPLACE WITH FLUTTER DOC LINK IF STILL RELEVANT ]
```
Comment (fuzzybinary) [inline content/en/real_user_monitoring/flutter/mobile_vitals.md:57]: Using Android docs for this is fine as it's an Android concept, and we only report ANRs on Android for Flutter.

**Review by fuzzybinary — APPROVED** [review]

**Comment by rtrieu** [comment]
@fuzzybinary @xgouchet Thanks for the review!

**Review by hestonhoffman — APPROVED** [review]
LGTM!

**Pull request merged**